### ISSUE TITLE
Problem installing on Ubuntu

### ISSUE THREAD

**Issue by almirnunes** [monicahq/monica#2320]
Hello,

I tried several times to install Monica on Ubuntu 18.04 following the instructions 
but in the end I always get this message:

|--------------------------------------------------------------------------
| Register The Auto Loader
|--------------------------------------------------------------------------
|
| Composer provides a convenient, automatically generated class loader for
| our application. We just need to utilize it! We'll simply require it
| into the script here so that we don't have to worry about manual
| loading any of our classes later on. It feels nice to relax.
|
*/

require __DIR__.'/../bootstrap/autoload.php';

/*
|--------------------------------------------------------------------------
| Turn On The Lights
|--------------------------------------------------------------------------
|
| We need to illuminate PHP development, so let us turn on the lights.
| This bootstraps the framework and gets it ready for use, then it
| will load up this application so that we can run it and send
| the responses back to the browser and delight our users.
|
*/

$app = require_once __DIR__.'/../bootstrap/app.php';

/*
|--------------------------------------------------------------------------
| Run The Application
|--------------------------------------------------------------------------
|
| Once we have the application, we can handle the incoming request
| through the kernel, and send the associated response back to
| the client's browser allowing them to enjoy the creative
| and wonderful application we have prepared for them.
|
*/

$kernel = $app->make(Illuminate\Contracts\Http\Kernel::class);

$response = $kernel->handle(
    $request = Illuminate\Http\Request::capture()
);

$response->send();

$kernel->terminate($request, $response);

**Comment by ehussong** [comment]
I'm experiencing the same exact issue.

**Comment by asbiin** [comment]
This may happen when php is not enabled on your http server.

**Comment by ehussong** [comment]
After following the installation instructions provided are there additional steps required?

**Comment by jfnklstrm** [comment]
I have exactly the same problem - tried to install it on vultr on ubuntu 18.10 18.04 and 16.04 but no matter how I do it I end up with the same error (like the one above). I would really appreciate some help because Im so stuck right now. I have also tried to set up monica using the instructions from github, from the instructions on vultr (https://www.vultr.com/docs/how-to-install-monica-on-ubuntu-18-04-lts). I also tried the instructions on (https://computingforgeeks.com/how-to-install-monica-personal-relationship-managerprm-on-ubuntu-18-04-lts/). No matter how I do it I end up with the same error!!

**Comment by wcypierre** [comment]
If you're following the default steps which is using Apache, the missing dependency is libapache2-mod-php7.1 which is used by Apache to load php (which is not included by the ondrej's repo)

tl;dr apt-get install libapache2-mod-php7.1 

Tested on ubuntu 18.04 proxmox lxc template

**Comment by wcypierre** [comment]
@jfnklstrm  in the computingforgeeks link, they provided the steps to install libapache2-mod-php but the package points to libapache2-mod-php7.**2** instead of libapache2-mod-php7.**1** and that is why it didn't worked out of the box.

**Comment by jfnklstrm** [comment]
Thank you wcypierre! Now it works, I also solved the problem with nginx on a ubuntu droplet - the issue in the steps on Vultr is that they are missing dependencies. For Monica to work you need to add php-gd, and php-bcmath + install unzip on the server.

Thank you very much for good and quick replies Pierre!!

**Comment by ehussong** [comment]
@wcypierre Thank you! That did it! Now working with the first set-up screen on Ubuntu 18.04 server thanks to your help. :)

**Comment by almirnunes** [comment]
@wcypierre THanks! Finally worked here!

**Comment by wcypierre** [comment]
@jfnklstrm  @ehussong @almirnunes  glad to be of help :)
